homework_2/task3.c

```diff
@@ -0,0 +1,76 @@
+#include <stdio.h>
+
+#define range 20000
+#define ar_ln 100
```

Comment:
```suggestion
#define MAX_LENGHT 100
```
Опять сокращение. И `define` принято большими буквами именовать.